jolie-cli/src/main/java/jolie/cli/CommandLineParser.java

```diff
@@ -339,12 +339,24 @@ public CommandLineParser( String[] args, ClassLoader parentClassLoader, Argument
 			} else if( "--connlimit".equals( argsList.get( i ) ) ) {
 				optionsList.add( argsList.get( i ) );
 				i++;
-				cLimit = Integer.parseInt( argsList.get( i ) );
+				try {
+					cLimit = Integer.parseInt( argsList.get( i ) );
+				} catch( Exception e ) {
+					System.out
```

Comment:
Mmh we should probably throw a CommandLineException, for consistency with the other error messages.

Reply:
Here I based myself to the parsing of `cellId`. https://github.com/jolie/jolie/blob/c714390e8aba275c2cb677320a1d5219bdd0704e/jolie-cli/src/main/java/jolie/cli/CommandLineParser.java#L440.

But of course we may change the behaviour also there.

Reply:
Okay I have fixed the two calls and push the refreshed commits.

Reply:
Super, thanks! :)